sdk/eventhub/mock-hub/package.json

```diff
@@ -43,29 +44,71 @@
   },
   "files": [
     "dist/",
-    "types/",
     "README.md",
     "License"
   ],
   "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/eventhub/mock-hub/README.md",
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser"
```

Comment:
we probably don't need `"browser"` dialect. I doubt it would work or be useful.

Reply:
good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/30634/commits/ceed4dc2e188f59a00bf35f26bcbe4a9d341e5a0